website/components/right-nav.vue

```diff
@@ -38,6 +45,7 @@ export default defineComponent({
       await nextTick()
       scrollContainer = document.querySelector('.el-scrollbar.page-component__scroll>.el-scrollbar__wrap.el-scrollbar__wrap--hidden-default')
       const content = document.querySelector('.content.element-doc.content')
+      if (!content) return
```

Comment:
Can we use `DOM ref` do to the exact same thing? Since you can add ref to `scrollbar` I think thats achievable

Reply:
Maybe it is a little difficult.

For the `scrollContainer`, because `scrollContainer` is innermost `div` of `scrollbar` component which in `component.vue`, though I  can provide the ref of the `scrollbar`, I also need query the innermost `div`.

For the `content`, this dom is in the component which generate by `md-loader`, the code in below:

```
const result = `
<template>
  <section class="content element-doc">
    ${output.join('')}
    <right-nav />
  </section>
</template>
${pageScript}
`
```

If use `ref` instead of querying dom, I need to modify `pageScript` which is a string value generate by code, I think it is unnecessary.